index.html

```diff
@@ -915,37 +1027,105 @@ <h4>Text</h4>
 
         <section>
           <h4>Text Language Source</h4>
-          <p>The <dfn>Text Language Source</dfn> property is an annotation indicating whether a <a>Text</a> object is
-            in the same language as the relevant part of the audio's
-            language (original), or if it is a representation in another language (translation):</p>
+          <p>The <dfn>Text Language Source</dfn> property is an annotation indicating the source language of
+            a <a>Text</a> object that represents content with an inherent language.
+            If it differs from the language of the <a>Text</a> object,
+            that implies that it is a representation in another language,
+            referred to as &quot;translation&quot;;
+            conversely if it is the same that implies that the language of the <a>Text</a> is <a>Original</a>.</p>
+          <p><a>Text Language Source</a> properties SHOULD NOT be present on <a>Text</a> objects
+            that represent content without an inherent language,
+            such as untranslated descriptions of a visual scene or
+            captions representing non-dialogue sounds.</p>
           <ul>
-            <li><dfn>Original</dfn> - the <a>Text</a> is any of:
+            <li><a>Text</a> is defined as <dfn>Original</dfn> if it is any of:
               <ol>
                 <li>the same language as the dialogue that it represents in the original programme audio;</li>
-                <li>a transcription of text visible in the original programme video, in the same language as that text;</li>
-                <li>representative of non-dialogue sound and is in the <a>Primary Language</a> of the <a>DAPT Script</a>;</li>
-                <li>representative of the scene in the original programme video, and is in the <a>Primary Language</a> of the <a>DAPT Script</a>.</li>
+                <li>a transcription of text visible in the programme video, in the same language as that text;</li>
+                <li>representative of non-dialogue sound;</li>
+                <li>a description of the scene in the programme video.</li>
               </ol>
             </li>
-            <li><dfn>Translation</dfn> - the <a>Text</a> is a representation of the Original language text in a different language.</li>
+            <li><dfn>Translation</dfn> - the <a>Text</a> is a representation of an <a>Original</a> <a>Text</a> object in a different language.</li>
           </ul>
           <aside class="note">The language of a <a>Text</a> object whose <a>Text Language Source</a> is <a>Original</a>
-            can be different to the document's <a>Primary Language</a>.
+            can be different to the document's <a>Default Language</a>.
+            Similarly it can be absent from the document's <a>Original Languages</a>.
```

Comment:
I wonder if the "Similarly ..." part should not be in its own, separate note. 
The first note is about the case of a source programme using multiple languages. We already have examples in the spec for that (EXAMPLE 11). 
The second note goes into a more specific use case, where that language is not in the original languages list. If not splitting the notes, at least, I would replace "Similarly" with "Furthermore" or "Additionally". I find "Similarly" confusing.

Reply:
Done in [01d26df](https://github.com/w3c/dapt/pull/179/commits/01d26dfa237f62aa96d98db149b0e3fe825bd630) - please check the new wording before resolving this conversation.